Dedupe MFA requirements in chapter V2 and update text

tghosth opened the pull request:
<!---
IMPORTANT NOTES:
- Changes should always be made only in the raw .md files and not in the CSV, JSON, XLSX, PDF, DOCX files, etc.
- Please do not open a pull request without first opening an associated issue.
- Please carry out all discussion in the associated issue only.
- Please refer to the following link for guidance on labeling contributions https://github.com/OWASP/ASVS/blob/master/CONTRIBUTING.md#standard-for-changes
-->

This Pull Request relates to issue #2587

tghosth commented:
(converted to draft to prevent accidental merge)

elarlang commented:
First I have question:
* 2.7.3 - maybe it is easier when this "once" stays in this requirement?

Tags to correct:
* 2.6.1 - covers 2.2.6, but 2.2.6 does not have matching tag "covered by 2.6.1" (2.8.4 is not updated to 2.6.1)
* 2.6.5 - modified, moved from (not added)
* 2.8.4 - deleted, merged to
* 2.8.3 - deleted, merged to

tghosth commented:
> First I have question:
> 
>     * 2.7.3 - maybe it is easier when this "once" stays in this requirement?
> 
> 
> Tags to correct:
> 
>     * 2.6.1 - covers 2.2.6, but 2.2.6 does not have matching tag "covered by 2.6.1" (2.8.4 is not updated to 2.6.1)
> 
>     * 2.6.5 - modified, moved from (not added)
> 
>     * 2.8.4 - deleted, merged to
> 
>     * 2.8.3 - deleted, merged to

Fixed tags and responded to your question here: https://github.com/OWASP/ASVS/issues/2587#issuecomment-2648646674

tghosth commented:
@elarlang are you ready to approve this now?

jmanico reviewed (approved):
This is going to collide with my "remove ETC" PR - but that is ok. Your PR is way more complex and I can adjust my branch after this is perged.
